Sources/Profile-View.php

```diff
@@ -446,13 +449,62 @@ function showAlerts($memID)
 
 	require_once($sourcedir . '/Profile-Modify.php');
 
+	// Are we opening a specific alert ?
+	if (!empty($_REQUEST['alert']))
```

Comment:
Do we need some kind of authority check here,
if he is allowed to open this?

Reply:
oh yes you're probably right, should check if the alert belongs to the member